config/_default/menus/menus.en.yaml

```diff
@@ -787,6 +787,94 @@ main:
     identifier: agent_guides
     parent: agent
     weight: 11
+  - name: Agent Commands
```

Comment:
@DavidWeid Do we still need required these changes to the `menus.en.yaml` file now that we're not listing individually in the nav? I think we can revert, correct?

Reply:
Some of the guide pages need to be listed in the menu for the state to properly display in the sidenav. I reviewed this another time this morning and updated the PR description to note which items are required or not (see additional notes). I've updated the file to remove unneeded subpages while keeping the required ones.